README.md

```diff
@@ -1,8 +1,10 @@
 # Request Generator
 
-This project provides a small web application that is capable of generating requests and displaying the CDS Hooks cards that are provided as a response. This project is written in JavaScript and runs in [node.js](https://nodejs.org/en/).
+This project provides a small web application that is capable of generating requests and displaying the CDS Hooks cards that are provided as a response. This app functionally serves as the frontend of the EHR it connects to, and is treated as part of the EHR system rather than a third party app. Typically, functionality provided by the Request Generator would be handled by the EHR in a production environment. The Request Generator makes it easy and simple to add CDS Hooks capability to SMART on FHIR compliant EHRs. 
```

Comment:
This sounds like a better description for the REMS SMART on FHIR app. The request generator was meant to be used for testing, not to be used alongside an EHR in production.